docs/pages/architecture/trustedclusters.mdx

```diff
@@ -1,65 +1,108 @@
 ---
 title: Teleport Trusted Clusters Architecture
-description: Deep dive into design of Teleport Trusted Clusters.
-h1: Trusted Clusters Architecture
+description: Deep dive into design of Teleport trusted clusters.
+h1: Trusted clusters architecture
 ---
 
-## Overview
-
 Teleport can partition compute infrastructure into multiple clusters. A cluster
 is a group of Teleport connected resources. Each cluster
 manages a set of certificate authorities (CAs) for its users and resources.
 
-Trusted Clusters allow the users of one cluster, the **root cluster**, to
+Trusted clusters allow the users of one cluster, the **root cluster**, to
```

Comment:
We capitalize "Trusted Cluster" in the [Core Concepts](https://goteleport.com/docs/core-concepts/#trusted-cluster) page and elsewhere in the docs. If we're going to lowercase it here, we should at least plan to do it everywhere.

Personally, I'm okay with "Trusted Cluster" being capitalized because it is a Teleport product, not a general computing term like "database".

What do you think?